### DISCUSSION TITLE
It's uptrace support for user roles?

### DISCUSSION THREAD

**Discussion by mrofisr** [uptrace/uptrace#252] [Q&A] [answered]
This is my first time trying uptrace, but I haven't found a way to assign roles to each user like (admin/user/etc), and how can I add a new user without editing uptrace.yaml?

Thankyou!

**Comment by vmihailenco** [comment, marked as answer]
This feature is only available in Enterprise Edition https://uptrace.dev/get/enterprise.html#organizations
